### DISCUSSION TITLE
Change where ASReview project  is saved.

### DISCUSSION THREAD

**Discussion by HiskeB** [asreview/asreview#998] [Q&A] [answered]
Question: 
For a review I use ASReview for a review, together with someone else. We do this in a common environment (digital research environment). We find it difficult that ASReview saves our project on a certain place in this environment (this place is not the common disc). We we're wondering whether it is possible to change the place where projects can be saved. In this way we can save it on the common disc, so that we can see each other changes. 

Thanks in advance!

HiskeB

**Comment by J535D165** [comment, marked as answer]
By default, all projects created in ASReview LAB are stored in the `.asreview` folder in the home folder (`~/.asreview`). One can change the location of the home folder by setting the `ASREVIEW_PATH` environment variable. There are various online resources to explain how you can set an environment variable, e.g. https://www.twilio.com/blog/2017/01/how-to-set-environment-variables.html.  

Ideally, the `ASREVIEW_PATH` path points to a basic disk and not to a mounted disk. (Mounted disks might face performance issues)

Please also keep in mind that this feature is not designed to be a solution for multi-user/multi-labeler use cases. For more ideas about collaborating, see https://github.com/asreview/asreview/discussions/975. Feel free to add your solution to the list as well.
